services/galaxy-toolshed/galaxy-toolshed-activity.service.js

```diff
@@ -0,0 +1,70 @@
+import Joi from 'joi'
+import { formatDate } from '../text-formatters.js'
+import { age as ageColor } from '../color-formatters.js'
+import BaseGalaxyToolshedService from './galaxy-toolshed-base.js'
+
+const repositoryRevisionInstallInfoSchema = Joi.array().items(
+  Joi.object({
+    create_time: Joi.string().required(),
+  }).required(),
+  Joi.object({}),
+  Joi.object({})
+)
+
+export default class GalaxyToolshedActivity extends BaseGalaxyToolshedService {
```

Comment:
Unless we anticipate additional metrics/badges being surfaced via this class, let's rename both the file and class name to be more explicit about the badge as opposed to the category, e.g. `GalaxyToolshedReleaseDate` or `GalaxyToolshedCreateDate`